infrastructure/quick-deploy/aws/all/parameters.tfvars

```diff
@@ -127,20 +132,34 @@ prometheus = {
 
 metrics_exporter = {
   node_selector = { "grid/type" = "Operator" }
+  extra_conf = {
+    MongoDB__AllowInsecureTls              = true
+    Serilog__MinimumLevel                  = "Information"
+    MongoDB__TableStorage__PollingDelayMin = "00:00:01"
+    MongoDB__TableStorage__PollingDelayMax = "00:00:10"
+  }
 }
 
-//parition_metrics_exporter = {
-//  node_selector = { "grid/type" = "Operator" }
-//}
+/*parition_metrics_exporter = {
+  node_selector = { "grid/type" = "Operator" }
+  extra_conf    = {
+    MongoDB__AllowInsecureTls           = true
+    Serilog__MinimumLevel               = "Information"
+    MongoDB__TableStorage__PollingDelayMin     = "00:00:01"
+    MongoDB__TableStorage__PollingDelayMax     = "00:00:10"
+  }
+}*/
 
 fluent_bit = {
   is_daemonset = true
 }
 
-
 # Logging level
 logging_level = "Information"
 
+# Job to insert partitions in the database
+job_partitions_in_database = {}
```

Comment:
As it is the default value, I am not sure if we really want to have in the tfvars.